git-keeper.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+#
+# git-keeper utility, used for backing up git repos to AWS S3
+# supported git providers github and gitlab
+# 
+# expecting ssh key for private repos and AWS S3 credentials configured
+# tested only on linux OS
+
+import os
+import sh
+import boto3
+import botocore
+import shutil
+import logging
+from datetime import datetime
+import sys
+import gnupg
+
+# bake some commands
+mirror = sh.git.clone.bake('--mirror')
+tar = sh.tar.bake('-cf')
+
+workdir = 'workdir'
+
+def upload2s3(repo_tar, DATE, object_name):
+    s3_client = boto3.client('s3')
+    try:
+        response = s3_client.upload_file(repo_tar, GIT_KEEPER_BUCKET, DATE + '/' + object_name)
+    except ClientError as e:
+        logging.error(e)
+        return False
+    return True
+
+def cleanwrkdir(workdir):
+    os.makedirs(workdir, exist_ok=True)
+    shutil.rmtree(workdir)
```

Comment:
you can change this to `shutil.rmtree(workdir, ignore_errors=True)` and remove the first `os.makedirs`.